File: src/main/java/com/sean/guess/Game.java
package com.sean.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int answer = random.nextInt(10) + 1;
        System.out.println(answer);
        int number = 0;
        while (answer != number) {
            System.out.println("enter your number");
            Scanner scanner = new Scanner(System.in);
            number = scanner.nextInt();
            if (answer < number) {
                System.out.println("lower");
            } else if (answer > number) {
                System.out.println("higher");
            } else {
                System.out.println("bingo , number is " + answer);
            }
        }
    }
}

File: src/main/kotlin/com/sean/Game.kt
package com.sean

import kotlin.random.Random

fun main() {
//    var scanner = (System.`in`)
    var answer = Random.nextInt(10) + 1
    var number = 0
    while (answer != number) {
        println("enter your number")
        number = readLine()!!.toInt()
        if (answer < number){
            println("lower")
        }else if (answer > number){
            println("higher")
        }else{
            println("bingo , answer is $answer")
        }
    }
}